contrib/mesos/pkg/minion/server.go

```diff
@@ -333,6 +336,10 @@ func (ms *MinionServer) AddExecutorFlags(fs *pflag.FlagSet) {
 
 	// hack to forward log verbosity flag to the executor
 	fs.Int32Var(&ms.logVerbosity, "v", ms.logVerbosity, "log level for V logs")
+	
+	// hacks to forward selected cadvisor flags to the executor
+	fs.StringVar(&ms.housekeepingInterval, "housekeeping_interval", ms.housekeepingInterval, "Interval between container housekeepings")
```

Comment:
if you have the prerequisites installed on your dev machine (docker,
docker-compose, etc) you can run `./contrib/mesos/ci/test-smoke.sh`

On Fri, Jan 29, 2016 at 1:23 PM, Lin Sun <notifications@github.com> wrote:

> In contrib/mesos/pkg/minion/server.go
> <https://github.com/kubernetes/kubernetes/pull/20331#discussion_r51295425>
> :
>
> > @@ -333,6 +336,10 @@ func (ms *MinionServer) AddExecutorFlags(fs *pflag.FlagSet) {
> >
> >  	// hack to forward log verbosity flag to the executor
> >  	fs.Int32Var(&ms.logVerbosity, "v", ms.logVerbosity, "log level for V logs")
> > +	
> > +	// hacks to forward selected cadvisor flags to the executor
> > +	fs.StringVar(&ms.housekeepingInterval, "housekeeping_interval", ms.housekeepingInterval, "Interval between container housekeepings")
>
> @jdef <https://github.com/jdef> Thanks - will rework. How can I trigger
> the smoke test myself locally? I have been trying to run:
> hack/build-go.sh
> hack/test-go.sh -- is this same as the smoke test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20331/files#r51295425>.
>